<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>457</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25,25, 40);</string>
  </property>
  <widget class="QComboBox" name="tipodedocumento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Estilo para el QComboBox */
QComboBox {
    color: rgb(231, 231, 231); /* Color del texto */
    font: 15pt &quot;Verdana&quot;; /* Fuente y tamaño del texto */
    border: 2px solid #4d4dff; /* Borde del ComboBox */
    border-radius: 10px; /* Radio de borde */
    padding: 5px; /* Relleno interno */
    background-color: rgb(25, 25, 40); /* Fondo del ComboBox */
    selection-background-color: darkgray; /* Color de fondo de la selección */
}

QComboBox::drop-down {
    border: none; /* Elimina el borde del botón desplegable */
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Cambia la imagen del botón desplegable si es necesario */
}

/* Estilo para el QComboBox cuando está desplegado */
QComboBox::drop-down:open {
    border-top-right-radius: 0; /* Ajusta el radio de borde superior derecho */
    border-bottom-right-radius: 0; /* Ajusta el radio de borde inferior derecho */
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Tipo de documento</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Carnet de identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Visa</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtNumeroDeDocumento">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Estilo para el QLineEdit */
QLineEdit {
    color: rgb(231, 231, 231); /* Color del texto */
    font: 15pt &quot;Verdana&quot;; /* Fuente y tamaño del texto */
    border: 2px solid #4d4dff; /* Borde del QLineEdit */
    border-radius: 10px; /* Radio de borde */
    padding: 0 8px; /* Relleno interno */
    background-color: rgb(25, 25, 40); /* Fondo del QLineEdit */
    selection-background-color: darkgray; /* Color de fondo de la selección */
}

QLineEdit:focus {
    border: 2px solid #4d4dff; /* Cambia el color del borde cuando el QLineEdit está enfocado */
}

QLineEdit:hover {
    border-bottom-color: #4d4dff; /* Cambia el color del borde inferior cuando el mouse está sobre el QLineEdit */
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Numero de documento</string>
   </property>
  </widget>
  <widget class="QComboBox" name="motivodelgiro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Estilo para el QComboBox */
QComboBox {
    color: rgb(231, 231, 231); /* Color del texto */
    font: 15pt &quot;Verdana&quot;; /* Fuente y tamaño del texto */
    border: 2px solid #4d4dff; /* Borde del ComboBox */
    border-radius: 10px; /* Radio de borde */
    padding: 5px; /* Relleno interno */
    background-color: rgb(25, 25, 40); /* Fondo del ComboBox */
    selection-background-color: darkgray; /* Color de fondo de la selección */
}

QComboBox::drop-down {
    border: none; /* Elimina el borde del botón desplegable */
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Cambia la imagen del botón desplegable si es necesario */
}

/* Estilo para el QComboBox cuando está desplegado */
QComboBox::drop-down:open {
    border-top-right-radius: 0; /* Ajusta el radio de borde superior derecho */
    border-bottom-right-radius: 0; /* Ajusta el radio de borde inferior derecho */
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Motivo del giro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios/Servicios tecnicos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Giros</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Estilo para el QLineEdit */
QLineEdit {
    color: rgb(231, 231, 231); /* Color del texto */
    font: 15pt &quot;Verdana&quot;; /* Fuente y tamaño del texto */
    border: 2px solid #4d4dff; /* Borde del QLineEdit */
    border-radius: 10px; /* Radio de borde */
    padding: 0 8px; /* Relleno interno */
    background-color: rgb(25, 25, 40); /* Fondo del QLineEdit */
    selection-background-color: darkgray; /* Color de fondo de la selección */
}

QLineEdit:focus {
    border: 2px solid #4d4dff; /* Cambia el color del borde cuando el QLineEdit está enfocado */
}

QLineEdit:hover {
    border-bottom-color: #4d4dff; /* Cambia el color del borde inferior cuando el mouse está sobre el QLineEdit */
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Monto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrarTransferenciaRegistro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>321</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(20, 200, 220);
    font: 87 17pt &quot;Arial Black&quot;;
    border: 2px solid green;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}

QPushButton:hover {
    color: rgb(20, 200, 220);
    background-color: rgba(20, 200, 220, 100);
}

QPushButton:pressed {
    background-color: rgba(20, 200, 220, 50);
}


</string>
   </property>
   <property name="text">
    <string>Registrar Transferencia</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
